common-custom-user-data-gradle-plugin/src/main/java/com/gradle/CustomBuildScanConfigExtras.java

```diff
@@ -0,0 +1,77 @@
+package com.gradle;
+
+import com.gradle.scan.plugin.BuildScanExtension;
+import org.gradle.api.Action;
+import org.gradle.api.Task;
+import org.gradle.api.invocation.Gradle;
+import org.gradle.api.tasks.testing.Test;
+
+import static com.gradle.Utils.*;
+
+final class CustomBuildScanConfigExtras {
+
+    static void configureBuildScan(BuildScanExtension buildScan, Gradle gradle) {
+        if (isCaptureEnabled("TestTaskConfiguration", true)) {
```

Comment:
I would set it to false by default.

Reply:
(different of today's behavior, I know)

Reply:
Sure. My intent was to preserve the existing functionality of the script.
Are you OK with Test task parallelism not being reported by default, too? (These are currently controlled by the same system property).

Reply:
I think it is ok. we can still tell people in the trial navigator to set these properties to true initially, and to turn them off if it is too much noise, but by then we already have a bseline.

Reply:
Done